### DISCUSSION TITLE
How can I add custom page icons?

### DISCUSSION THREAD

**Discussion by zcraber** [JulianPrieber/littlelink-custom#512] [Questions] [answered]
Is it possible to add custom page icons? For e.g., I want to add Telegram 
![image](https://github.com/LinkStackOrg/LinkStack/assets/61133303/9a061d99-2220-4e00-8c82-38ca0de5d69f)

**Comment by JulianPrieber** [comment]
https://github.com/LinkStackOrg/LinkStack/commit/972e7c7c96dd7ba6c9cde87ffe378c68a7fe337b

**Reply by zcraber** [reply]
Facebook is already there, but Telegram is missing. I'm referring to this section: 

![image](https://github.com/LinkStackOrg/LinkStack/assets/61133303/a0a74e64-8460-453e-8294-1e02546bd867)

**Reply by JulianPrieber** [reply]
Right, me too. We did not include messengers in our initial batch of icons.
The commit shown you how you can easily add the telegram icon to your page.

**Reply by zcraber** [reply]
Oh, ok. I'm using the Docker version, so how do I add this? Sorry, I'm new to this.

**Reply by JulianPrieber** [reply]
Ah then its a bit harder. One sec pls.

**Reply by JulianPrieber** [reply]
Do you already have links on your page?

**Reply by zcraber** [reply]
Yes, already have links.

**Reply by JulianPrieber** [reply]
If you're comfortable sharing those links, please go to `your.page/studio/profile` and click on export links only and attach the file here.

**Reply by zcraber** [reply]
Here it is.

[links-links.digitalmalayali.in-2023-07-30_15-59-24.zip](https://github.com/LinkStackOrg/LinkStack/files/12209628/links-links.digitalmalayali.in-2023-07-30_15-59-24.zip)

**Comment by JulianPrieber** [comment, marked as answer]
[links-localhost_880-2023-07-30_16-03-20.zip](https://github.com/LinkStackOrg/LinkStack/files/12209637/links-localhost_880-2023-07-30_16-03-20.zip)

On the same page, if you import this file, you should have the telegram icon on your page.

We might add an official way to do this later on..

**Reply by zcraber** [reply]
Thank you! An official way would be really useful.
